frontend/src/components/PLAYGROUND/Pages/ProjectsSandbox/index.tsx

```diff
@@ -1,5 +1,14 @@
 import { Table } from "components/PLAYGROUND/Table";
 
 export const ProjectsSandbox = () => {
-  return <Table apiEndPoint={"projects"} />;
+  const handleRowDoubleClick = () => {};
+  // return <Table apiEndPoint={"projects"} />;
```

Comment:
is there a reason for leaving this comment?
Please ensure if any comments are left in, there is a todo to indicate they need to be removed in future.

Reply:
I deleted that file, could you please re-review/approve for the file removal?